ddtrace/internal/ci_visibility/encoder.py

```diff
@@ -199,7 +199,7 @@ def _convert_span(self, span, dd_origin):
         converted_span = {
             "test_session_id": int(span.get_tag(SESSION_ID) or "1"),
             "test_suite_id": int(span.get_tag(SUITE_ID) or "1"),
-            "files": json.loads(span.get_tag(COVERAGE_TAG_NAME))["files"],
+            "files": json.loads(str(span.get_tag(COVERAGE_TAG_NAME)))["files"],
```

Comment:
How is this related to moving the module?

Reply:
https://github.com/DataDog/dd-trace-py/pull/8274#discussion_r1482034544

Reply:
Better to leave a comment in code?

Reply:
Nah it's a reasonable change - I just had a hard time understanding how it's related.